pom.xml

```diff
@@ -243,6 +243,7 @@ under the License.
     <module>helix-core</module>
     <module>helix-admin-webapp</module>
     <module>helix-rest</module>
+    <module>helix-lock</module>
```

Comment:
Question: My understanding is that this distributed lock is supposed to be a general module that does not assume the use of Helix. Should we call this something like zk-distributed-lock?